performance-tests/02-no-filters/run.sh

```diff
@@ -13,7 +13,13 @@ SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 CFG=02-no-filters/config.yaml
 ENDPOINT=kroxylicious:9092
 
-KROXYLICIOUS_CONFIG=${CFG} runDockerCompose up --detach --wait kroxylicious
+KROXYLICIOUS_CONFIG=${CFG} runDockerCompose up --detach
+RUNNING=$(docker inspect -f {{.State.Running}} kroxylicious)
+echo "running = $RUNNING"
+until [ "$RUNNING" == "true" ]; do
```

Comment:
The logic looks flawed.  RUNNING is being assigned once (before tte loop) then you check the same value over and over.  You need to update running within the loop.   Do .. while might be a more a suitable iteration construct.

Reply:
Yes, the changes were only testing if it works somehow. That's why I set it to Draft

Reply:
> Yes, the changes were only testing if it works somehow. 
you have a race.  It'll work if line 17 yields running , otherwise it will loop forever.

Reply:
I know. As I mention this is only intended for testing what could we do to wait, but no matter if it loops forever or not, it is not working as expected.
I haven't time to retake this PR